### PR TITLE
Add fpbioimage to OMERO features view page

### PR DESCRIPTION
**Pull request by hflynn**
Added to the main section as we've bundled and distributed it ourselves (AFAIK) but happy to move it to the community section at the bottom if that's more appropriate.

### CODE REVIEW

**Comment by hflynn** [comment]
Staged at https://snoopycrimecop.github.io/www.openmicroscopy.org/omero/features/view/

File: img/icons/omero-features-icons_fpbioimage.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg id="fpbioimage" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><defs><style>.cls-1{fill:#03a9f4;}.cls-1,.cls-2,.cls-3{stroke:#263238;stroke-miterlimit:10;stroke-width:2px;}.cls-2{fill:#009688;}.cls-3{fill:#ef5350;}</style></defs><title>omero-features-icons</title><polygon class="cls-1" points="63.91 15.56 44.86 27.21 63.92 38.87 83.41 27.42 63.91 15.56"/><polygon class="cls-2" points="85 32 66 43.59 66 66.22 85 55.33 85 32"/><polygon class="cls-2" points="41 111.34 62 98.75 62 75.67 41 87.98 41 111.34"/><polygon class="cls-3" points="62 43.57 43 31.72 43 55.16 62 66.22 62 43.57"/><polygon class="cls-1" points="39.28 60.05 17.46 71.6 37.31 83.26 58.62 71.09 39.28 60.05"/><polygon class="cls-3" points="15 99.74 34 111.33 34 87.96 15 76.11 15 99.74"/><polygon class="cls-3" points="66 98.74 88 111.33 88 87.96 66 75.39 66 98.74"/><polygon class="cls-1" points="69.64 70.88 90.87 83.26 110.81 71.81 89.03 59.96 69.64 70.88"/><polygon class="cls-2" points="94 111.34 113 99.74 113 76.39 94 87.98 94 111.34"/></svg>
```
Comment (jburel) [inline img/icons/omero-features-icons_fpbioimage.svg]: Add extra line

Comment (hflynn) [reply img/icons/omero-features-icons_fpbioimage.svg]: Done

**Comment by hflynn** [comment]
https://snoopycrimecop.github.io/www.openmicroscopy.org/omero/features/view/ updated - happier with this wording @jburel ?

We can leave this on hold if you'd rather get a more advanced version on PyPI before we promote it, I just noticed that some folks are already using it from Twitter so it seemed silly not to have it here.

**Comment by jburel** [comment]
I will not keep on hold since we do not have any control of the release of FPBioImage
so we should still expose

**Comment by hflynn** [comment]
Hopefully the wording is better now then

**Comment by jburel** [comment]
I think that's better

**Comment by hflynn** [comment]
@sbesson / @joshmoore if you're happy with this too, we can merge and get this live when someone has time (I was going to say with the feature header change but then I noticed that's already live - thanks)

**Comment by hflynn** [comment]
Merging at EOB if no zeto from @sbesson

**Pull request merged**